package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "contentful-metalsmith",
+  "version": "0.8.1",
```

Comment:
ah, that shouldn't have been part of the pull request, it was just so we could get npm pulling from our repository for now as we needed this functionality.